src/interaction/index.ts

```diff
@@ -8,9 +8,9 @@ import * as unload from "@src/interaction/unload";
 import * as visibility from "@src/interaction/visibility";
 
 export function start(): void {
-    click.start();
-    input.start();
```

Comment:
The different between reset and start here is very a hard to understand.

Reply:
Agreed - fixed it. Now everything has a start & end. Leads to more bytes but we can optimize that later. Consistency is better for now.